std/experimental/logger/filelogger.d

```diff
@@ -200,3 +227,20 @@ class FileLogger : Logger
     assert(tl !is null);
     stdThreadLocalLog.logLevel = LogLevel.all;
 }
+
+@safe unittest
+{
+    import std.file : exists, rmdirRecurse;
+
+    string logpath = "super/long/path/";
```

Comment:
FYI: `deleteme` just returns a string sth. like `/tmp/deleteme.dmd.unittest.pid1234` - it can be used to construct temporary folders as well.